### PR TITLE
FIX 403 Error

### LINKED ISSUE #7: TKK generated
Hey.

Did you manage to figure out how TKK is generated?

`var code = res.body.match(/TKK='.*?';/g);`

**TKK=** not found in HTML `https://translate.google.com`

tkk

Where does TKK come from?

Maybe this: `tkk:'434303.3040823967'`

### PR DESCRIPTION
**Pull request by Adizbek**
Removed proxy support, as they make more complexity. Fixed token generator.

I am still testing, I haven't get any 403 error yet

Closes #7

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ async function translate(text, options) {
 
     // Check if a lanugage is in supported; if not, throw an error object.
     let error;
-    [ options.from, options.to ].forEach((lang) => {
+    [options.from, options.to].forEach((lang) => {
```
Comment (ZuBB) [inline src/index.js]: probably you should revert this whitespace changes. in this PR we expect to see changes related to subject of this PR only.

any whitespace/formatting issues may be addressed in separate PR

File: src/tokenGenerator.js [hunk 2]
```diff
@@ -76,22 +78,29 @@ function updateTKK() {
...
 
+        // if (code) {
+        //   eval(code[0]);
+        //   /* eslint-disable no-undef */
+        //   if (typeof TKK !== 'undefined') {
+        //     window.TKK = TKK;
+        //     config.set('TKK', TKK);
+        //   }
+        //   /* eslint-enable no-undef */
+        // }
```
Comment (ZuBB) [inline src/tokenGenerator.js]: there is no point in keeping commented old code. either remove it or as much as possible from ol approach

**Comment by Adizbek** [comment]
As you can see I labelled it as WIP, later I will do code reformating, I am
waiting for confirmation from others.

File: src/tokenGenerator.js [hunk 3]
```diff
@@ -76,22 +78,29 @@ function updateTKK() {
       else {
         let res = await got('https://translate.google.com');
 
-        const code = res.body.match(/TKK=(.*?)\(\)\)'\);/g);
+        const code = res.body.match(/tkk:'\d+.\d+'/g);
```
Comment (ZuBB) [inline src/tokenGenerator.js]: * is `tkk` really become lowercased?
* there is no need in using `g` in regex if we later use only 1st match (`code[0].split...`)
* imo there is point in adding example of response body and highlighting of meaningful part so other become aware of exact thing we are interested in

Comment (Adizbek) [reply src/tokenGenerator.js]: image

Just go to and view source of [https://translate.google.com/](https://translate.google.com/)
I am trying to extract value of **tkk**

Comment (k3rn31p4nic) [reply src/tokenGenerator.js]: So, the tkk now being lowercase was the issue for us.
And btw, for the regex, you can do a case insensitive match so that it won't matter what case it's in.

Comment (Adizbek) [reply src/tokenGenerator.js]: Yes was lowercased, moved into json, as I can see from code, before it was in other format.

**Comment by Adizbek** [comment]
I think I've finished, just check both GET and POST requests. should work as intended.

**Comment by ZuBB** [comment]
hey @k3rn31p4nic how about releasing new version with this change?

**Comment by k3rn31p4nic** [comment]
Gonna do that after I test it with the application where I use it regularly to see if I need to do any changes before releasing.

**Comment by Adizbek** [comment]
Some new issue
```js
Error while translating
{ HTTPError
    at translate (/opt/translator/node_modules/@k3rn31p4nic/google-translate-api/src/index.js:143:19)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:182:7)
  name: 'HTTPError',
  statusCode: 429,
  statusMessage: 'Too Many Requests' }
```

**Comment by ZuBB** [comment]
Hi @k3rn31p4nic!

do you have any news on your testing?

a bit offtop: just noticed that this package does not have minimum required version of node to run. I made a quick walk through code and found `conts`/`let` and `async`/`await` which gived us 'v7.10.1'  but probably you know better this kind of stuff. Would be nice to have it in `package.json`

**Comment by k3rn31p4nic** [comment]
> do you have any news on your testing?

Yeah, it works.

> just noticed that this package does not have minimum required version of node to run.

I'll set the node engine version and make a release soon.

**Comment by k3rn31p4nic** [comment]
Released in v1.1.0 :tada:

**Comment by NhocKun** [comment]
> Some new issue
> 
> ```js
> Error while translating
> { HTTPError
>     at translate (/opt/translator/node_modules/@k3rn31p4nic/google-translate-api/src/index.js:143:19)
>     at <anonymous>
>     at process._tickCallback (internal/process/next_tick.js:182:7)
>   name: 'HTTPError',
>   statusCode: 429,
>   statusMessage: 'Too Many Requests' }
> ```

Same here, anyone fixed this ?
